internal/statestore/redis_indices.go

```diff
@@ -44,6 +44,12 @@ func extractIndexedFields(cfg config.View, t *pb.Ticket) map[string]float64 {
 		switch v.Kind.(type) {
 		case *structpb.Value_NumberValue:
 			result[rangeIndexName(attribute)] = v.GetNumberValue()
+		case *structpb.Value_BoolValue:
```

Comment:
Is the expectation to set this to false explicitly if false is needed? Or does absence equate automatically to false? 
Just to explicitly clarify:
set foo --> false on ticket:
filter:foo value: false returns the ticket in results. 

Do not set foo at all in Ticket. Do we expect filter:foo, value:false to return the ticket or not?
We should document the expected behavior.

Reply:
All filters will not include any tickets where the field being filtered on is absent.  This is a good question though.  I updated the messages.proto example to be false, so it's more obvious.